taskipy/task.py

```diff
@@ -19,6 +20,23 @@ def command(self):
     def description(self):
         return self.__task_description
 
+    @property
+    def use_vars(self):
+        return self.__task_use_vars
+
+    def __extract_task_use_vars(self, task_toml_contents: object) -> bool:
+        if isinstance(task_toml_contents, str):
+            return False
+        if isinstance(task_toml_contents, dict):
+            try:
+                value = task_toml_contents['use_vars']
+            except KeyError:
+                return False
```

Comment:
Just a suggestion (not a must ;) )
```suggestion
            value = task_toml_contents.get('use_vars', False)
```